### PR TITLE
Remove .git-blame-ignore-revs to avoid inaccurate blame on github

### PR DESCRIPTION
**Pull request by sharwell**
As promised, I avoided sending this PR until the first time it was clear without looking that the blame view was inaccurate and unable to be corrected by a per-user setting.

image

### CODE REVIEW

**Review by JoeRobich — APPROVED** [review]
Ideally GitHub would support ignoring these commits optional in their Blame view.

**Comment by sharwell** [comment]
Waiting until 29 March to merge this to see if we can get feedback to GitHub first.

**Pull request closed without merging**
